app/modules/ads/index.js

```diff
@@ -548,12 +549,10 @@ class Ads {
    * @private
    */
   trackIdentityLibraryLoadTime() {
-    const { identityLibraryLoadedEvent, eventService } = window.Wikia.adEngine;
-
-    eventService.communicator.actions$.pipe(
-      ofType(identityLibraryLoadedEvent),
-    ).subscribe((props) => {
-      pageTracker.trackProp('identity_library_load_time', props.loadTime.toString());
+    communicationService.addListener((action) => {
+      if (isType(action, '[AdEngine] Identity library loaded')) {
```

Comment:
Why `identityLibraryLoadedEvent` is not used anymore? It's exported in AdEngine still.

Reply:
I try to prepare code for AdEngine transition in the future. Less coupling = better. It is not necessarily important in this task but since I went through all instances of post-quecast I thought it will make future work simpler.

Reply:
Got it 👍